src/server/editorServices.ts

```diff
@@ -1197,6 +1200,11 @@ namespace ts.server {
         private forEachConfigFileLocation(info: ScriptInfo,
             action: (configFileName: NormalizedPath, canonicalConfigFilePath: string) => boolean | void,
             projectRootPath?: NormalizedPath) {
+

```

Comment:
Can you also add check in ```openClientFileWithNormalizedPath``` to skip config project open check ```(!project && !this.syntaxOnly)```
https://github.com/Microsoft/TypeScript/blob/master/src/server/editorServices.ts#L2007

Reply:
I made this change in https://github.com/Microsoft/TypeScript/pull/23128.